aggregate_spatial_window.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "aggregate_spatial_window",
+  "summary": "Zonal statistics for rectangular windows",
+  "description": "Aggregates statistics over the spatial dimensions. This process passes a list of values to the reducer.\n\nThe results are computed dividing the input X/Y grid into x_size/y_size non-overlapping windows and applying the reducer process to each window.\nIf X and Y are not multiple of x_size/y_size NaNs padding before the computation is applied.",
+  "categories": [
+    "cubes",
+    "aggregate & resample"
+  ],
+  "parameters": [
+    {
+      "name": "data",
+      "description": "A raster data cube.",
```

Comment:
Does it have constraints? It seems like the input raster data cube must have only two dimensions? x and y...

Reply:
In the implementation I am working on, it just needs to have x and y, if it is a timeseries then it will be applied on the full timeseries and return another timeseries. So the datacube dimensions could be (time,x,y) or (x,y) or (band,x,y) or (time,band,x,y) and so on. But we can also choose to limit it to 2d datacubes to ease the implementation.

Reply:
Let's discuss later in the meeting.

Reply:
This still needs clarification

Reply:
what about:
 "description": "A raster data cube with two spatial dimensions and arbitrary number of bands and times. If the datacube has multiple bands the process will be applied to all of them. Same for a timeseries."

Sorry I'm not good in writing this kind of descriptions.